Dockerfile

```diff
@@ -1,40 +1,49 @@
+FROM ubuntu:focal AS base-dependencies
+ENV PATH="/root/.local/bin:${PATH}"
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG C.UTF-8
+ENV C_INCLUDE_PATH=/usr/include
+# Install python and import python dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes \
```

Comment:
I moved to be shared between pip install and the runtime image because Pillow dependencies are needed during pip install, so this makes the dependencies more maintainable